qemu/tests/balloon_check.py

```diff
@@ -210,6 +210,9 @@ def _memory_check_after_sub_test():
             elif should_quit == 0:
                 expect_mem = self.ori_mem
 
+            if sub_type == "boot":
```

Comment:
How about fix this issue like 
```
sleep_before_check = int(params.get("sleep_time_before_check", 0))
mmem, gmem = utils_misc.wait_for(_memory_check_after_sub_test,
timeout, sleep_before_check, 5, msg)
```
And add "sleep_time_before_check = 90" if sub_type == boot in  test configuration